activerecord/test/models/mikan.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Mikan < ActiveRecord::Base
+end
+
+class Unshu < Mikan
```

Comment:
```suggestion
class Unshiu < Mikan
```
nit: 
I think the Romanization of Citrus unshiu is “Unshiu”, not “Unshu”🍊
ref) https://en.wikipedia.org/wiki/Citrus_unshiu